services/grafana/auth_server.go

```diff
@@ -131,7 +131,7 @@ func (s *AuthServer) ServeHTTP(rw http.ResponseWriter, req *http.Request) {
 
 	if err := extractOriginalRequest(req); err != nil {
 		s.l.Warnf("Failed to parse request: %s.", err)
-		rw.WriteHeader(400)
+		rw.WriteHeader(http.StatusBadRequest)
```

Comment:
@AlekSi But `http.StatusBadRequest` is same 400 `int` but defined as constant. Nothing really changed, or I missed something?

Reply:
Is 400 less explicit than `http.StatusBadRequest`, 200 – less than `http.StatusOK` in the context of HTTP protocol responses? Can't say I necessary agree with that in general (for example, I remember the difference between 301/302/303/307/308 by codes, but constant names are… not very descriptive 😄), but that's a different topic.

But in that particular case 400/200 is much explicit since nginx configuration uses numbers (and can't use named constants), and that code is tightly coupled with it.

So let's not use named constants for status codes there.

Reply:
Ok, I get it. Will mute linter)